mappings/net/minecraft/server/network/EntityTrackerEntry.mapping

```diff
@@ -21,14 +24,22 @@ CLASS net/minecraft/unmapped/C_yagtbmmk net/minecraft/server/network/EntityTrack
 		ARG 4 alwaysUpdateVelocity
 		ARG 5 receiver
 	METHOD m_acqaqxfb sendPackets (Lnet/minecraft/unmapped/C_mxrobsgg;Ljava/util/function/Consumer;)V
+		ARG 1 player
 		ARG 2 sender
 	METHOD m_acwzlrjk sendSyncPacket (Lnet/minecraft/unmapped/C_zyjtrjrl;)V
 		COMMENT Sends a packet for synchronization with watcher and tracked player (if applicable)
 		ARG 1 packet
 	METHOD m_dlxmtkps syncEntityData ()V
 		COMMENT Synchronizes tracked data and attributes
+	METHOD m_eszryakz getNonUniqueEntries (Ljava/util/List;Ljava/util/List;)Ljava/util/stream/Stream;
```

Comment:
```suggestion
	METHOD m_eszryakz getDifference(Ljava/util/List;Ljava/util/List;)Ljava/util/stream/Stream;
```
The stream returned contains all elements in `list2` *not* contained in `list1`, so the original name is backwards. 

Additionally, I think "unique" strongly implies that the method filters out duplicate entries, which it doesn't. 

I have two minor reservations about `getDifference` though:
1. the paramaters are `Lists`, not java `Set`s [but I think it's pretty clear how 'difference' would be applied to lists]
2. the returned stream is `list2` minus `list1`, not the more intuitive `list1` minus `list2` [but I doubt there's a concise-enough way to phrase that to put it in the name, maybe something for javadoc]

Reply:
I was treating this as strictly a utility method, but I properly checked the moj and its usages and it's only used to find removed passengers of an entity. I've named it `streamRemovedPassengers` to reflect this!